glyphs-reader/src/font.rs

```diff
@@ -325,6 +326,40 @@ fn custom_param<'a>(
     None
 }
 
+fn custom_params(other_stuff: &BTreeMap<String, Plist>) -> Option<&Vec<Plist>> {
+    let custom_params = other_stuff.get("customParameters");
+    custom_params.as_ref()?;
+    let Some(Plist::Array(custom_params)) = custom_params else {
+        warn!("customParameters isn't an array, omg omg omg");
+        return None;
+    };
+    Some(custom_params)
+}
+
+fn custom_param<'a>(
+    other_stuff: &'a BTreeMap<String, Plist>,
+    key: &str,
+) -> Option<(usize, &'a Plist)> {
+    let Some(custom_params) = custom_params(other_stuff) else {
+        warn!("customParameters isn't an array, omg omg omg");
+        return None;
+    };
+
+    let name_key = "name".to_string();
+    for (idx, custom_param) in custom_params.iter().enumerate() {
+        let Plist::Dictionary(dict) = custom_param else {
```

Comment:
sorry but custom parameters can well be strings. They can be anything a plist can be :)

Reply:
ignore me, I misunderstood the code, of course custom  parameter is a {"name": <string>, "value": Plist} dict